### DISCUSSION TITLE
ERROR Transfer event TRR DMA ptr not part of current TD ep_index 4 comp_code 13

### DISCUSSION THREAD

**Discussion by Pme1159** [zwave-js/zwave-js#6492] [Request Support / Investigate Issue] [answered]
Z-Wave JS UI version: **2.x.x**

Hello, the version 1.15.5 of Zwave JS UI runs without any problem on my Home Assistant installed under an Oracle virtual box version 7.0 running on Windows 10 on a PC dedicated to my domotic.

But as soon as I try to update the version 1.15.5 to the last version available in Home Assistant (v 3.0.0) I get : 
- the following error message at the level of Zwave JS UI : .
![image](https://github.com/zwave-js/zwave-js-ui/assets/59808783/5737815b-da84-4515-8eab-6dcbec9dfd50)
- the message here above in title at the level of the "terminal" : Error transfer ...

This problem occured already with the versions 2.x.x of Zwave JS UI.

I use the Gen 5 Plus Aeotec key.

The problem was discussed under the forum HACF ant it appears that I'm not alone to get this issue.
(https://forum.hacf.fr/t/error-transfer-event-trr-dma-ptr-not-part-of-current-td-ep-index-4-comp-code-13/27047/22)

I thank you in advance for your help.

**Comment by robertsLando** [comment]
Check: https://github.com/zwave-js/node-zwave-js/issues/6341

**Comment by Pme1159** [comment]
Hello, I thank you for your answer. I applied the 1st, the 2nd and the 6th solutions. For the 4 and 5, I don't know what to do with my configuration. It must be said that compared to you, I am a Boeotian in computer science and it is not easy for me to understand what I have to do in my case.
After applying the solutions hereabove, I retried to update Zwave JS UI but with the same issues. I also tried to use a port USB 3 instead of the USB 2 on which the key is plugged, but without success.
Pascal

**Comment by robertsLando** [comment]
Tried to disable soft reset too and that doesn't work neighter?

If so please make a [driver log](https://zwave-js.github.io/zwave-js-ui/#/troubleshooting/generating-logs?id=driver-logs), loglevel `debug` and attach it here as a file (drag & drop into the text field).

**Reply by Pme1159** [reply, marked as answer]
Hello, it is OK with the soft reset disabled. 
But now, I have the same problem with my zigbee key !

**Reply by robertsLando** [reply]
cc @AlCalzone

**Reply by AlCalzone** [reply]
Can't help with Zigbee, sorry.
